lefthook.yml

```
@@ -49,14 +48,21 @@ pre-push:
           exit 1
         fi
 
+    - name: field-alignment
+      glob: "**/*.go"
+      root: "umh-core"
+      run: |
+        make betteralign
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)